examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -18,7 +18,7 @@
   "package": [
     {
       "pathRelativity": "relativeToZap",
-      "path": "../../../src/app/zap-templates/zcl/zcl-with-test-extensions.json",
+      "path": "../../../src/app/zap-templates/zcl/zcl.json",
```

Comment:
This change is not right... It will cause failures.

Reply:
Removed .zap file from the PR